docs/functions.rst

```diff
@@ -200,9 +200,9 @@ at the beginning of the wrapped code.
 
 C helper functions
 
-``ShroudStrCopy(char *a, int la, const char *s)``
-    Copy *s* into *a*, blank fill to *la* characters
-    Truncate if *a* is too short.
+``ShroudStrCopy(char *dest, int la, const char *src, int ls)``
+    Copy *src* into *dest*, blank fill to *la* characters
+    Truncate if *dest* is too short to hold all of *src*.
```

Comment:
Is it worth mentioning that you are truncating to one less than the length of dest?  (I assume)

Reply:
dest will not have a terminating NULL since the result is intended to be a Fortran CHARACTER variable, so it will copy at most ndest characters.
I added a comment to be clear about the lack of NULL.